src/pages/billing/Project.js

```diff
@@ -384,6 +384,7 @@ const ProjectDetail = ({ billingProject, reloadBillingProject, billingAccounts,
         h(Link, {
           tooltip: 'Remove Billing Account',
           style: { marginLeft: '0.5rem' },
+          disabled: billingProject.billingAccount === 'null',
```

Comment:
As expected, I'm seeing some billing projects with the button disabled if the billing account is "No Access" (which is maybe not the best name for "no billing account", but that's separate issue) 

However, I'm also seeing that if I add and remove a billing account (or just remove an existing billing account) it will not disable the button, even after refreshing the page.